helm_deploy/laa-dces-drc-integration/templates/deployment.yaml

```diff
@@ -86,6 +86,11 @@ spec:
             - secretRef:
                 name: dces-datasource
                 optional: false
+            {{/* All the keys defined in config-variables secret mentioned below will become environment variables, but only if
+             same named variables haven't already been defined via the env: tag, here or in _environment.tpl */}}
+            - secretRef:
+                name: config-variables
+                optional: true
```

Comment:
Does putting this before you call the `env-vars` template not mean that the keys in `config-variables` can't override the values in the `env-vars` template (`_environment.tpl`). Isn't the point that you can override environment values by updating the Secret?

Reply:
Discussed with Vid: envFrom can never override env, so order is irrelevant. This is why he's using the application.yaml default instead of env as default.